_posts/2020-08-11-release-1.10.2.md

```diff
@@ -0,0 +1,199 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17800'>FLINK-17800</a>] -         RocksDB optimizeForPointLookup results in missing time windows
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17801'>FLINK-17801</a>] -         TaskExecutorTest.testHeartbeatTimeoutWithResourceManager timeout
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17809'>FLINK-17809</a>] -         BashJavaUtil script logic does not work for paths with spaces
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17822'>FLINK-17822</a>] -         Nightly Flink CLI end-to-end test failed with &quot;JavaGcCleanerWrapper$PendingCleanersRunner cannot access class jdk.internal.misc.SharedSecrets&quot; in Java 11 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17870'>FLINK-17870</a>] -         dependent jars are missing to be shipped to cluster in scala shell
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17891'>FLINK-17891</a>] -          FlinkYarnSessionCli sets wrong execution.target type
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-17959'>FLINK-17959</a>] -         Exception: &quot;CANCELLED: call already cancelled&quot; is thrown when run python udf
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18008'>FLINK-18008</a>] -         HistoryServer does not log environment information on startup
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18012'>FLINK-18012</a>] -         Deactivate slot timeout if TaskSlotTable.tryMarkSlotActive is called
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18035'>FLINK-18035</a>] -         Executors#newCachedThreadPool could not work as expected
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18045'>FLINK-18045</a>] -         Fix Kerberos credentials checking to unblock Flink on secured MapR
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18048'>FLINK-18048</a>] -         &quot;--host&quot; option could not take effect for standalone application cluster
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18097'>FLINK-18097</a>] -         History server doesn&#39;t clean all job json files
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18168'>FLINK-18168</a>] -         Error results when use UDAF with Object Array return type
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18223'>FLINK-18223</a>] -         AvroSerializer does not correctly instantiate GenericRecord
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18241'>FLINK-18241</a>] -         Custom OptionsFactory in user code not working when configured via flink-conf.yaml
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18242'>FLINK-18242</a>] -         Custom OptionsFactory settings seem to have no effect on RocksDB
```

Comment:
it does not tell user explicitly that any previous program depending on 'RocksDBOptionsFactory' or extending `DefaultConfigurableOptionsFactory` might be broken as we remove `RocksDBOptionsFactory#createDBOptions(DBOptions)`  and `RocksDBOptionsFactory#createColumnOptions(ColumnFamilyOptions)` interfaces to fix this issue.

Reply:
A new note section is added to show the possible incompatible changes of FLINK-18242.
Would you take another look?

Reply:
Thanks for your fix, LGTM now.